src/test/java/io/camunda/zeebe/bpmnassert/testengine/PrintRecordStreamExtension.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.bpmnassert.testengine;
+
+import java.lang.reflect.Field;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.TestWatcher;
+import org.junit.platform.commons.util.ReflectionUtils;
+
+class PrintRecordStreamExtension implements TestWatcher {
```

Comment:
:question: Why is this in the test folder? I would expect it in the source folder

Reply:
I copied this from EZE. I think it makes sense because this is only used for the unit tests of the engine. EZE has a similar construct in the extension and I think we should add this to our extension as well.

When we split this project into multiple modules our engine will need this anyway as it shouldn't depend on the module that contains our extension.